lib/voyage/templates/api_base_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class BaseApiController < ApplicationController
+    # Disable CSRF protection for API calls
+    protect_from_forgery with: :null_session
+
+    # Disable cookie usage
+    before_action :destroy_session
+
+    # Handle objects that aren't found
+    rescue_from ActiveRecord::RecordNotFound, with: :not_found
+
+    def respond_with_errors(object)
```

Comment:
I use this method to have our error responses match the JSON API spec and simplify error handling. Whenever an error occurs in an API controller, we can just call respond_with_errors(@model_instance) and get the proper error response.